### ISSUE TITLE
testExamplesJaxprDoc in Understanding Jaxprs

### ISSUE THREAD

**Issue by 8bitmp3** [google/jax#3678]
@mattjj @j-towns In [Understanding Jaxprs](https://jax.readthedocs.io/en/latest/jaxpr.html), there's a reference to code examples in `jax/tests/api_test::JaxprTest.testExamplesJaxprDoc` but @mattjj mentioned you may want to delete/have already deleted `testExamplesJaxprDoc` here https://github.com/google/jax/pull/2994#issuecomment-625490111. Shall we get rid of the link in the [`jaxpr.rst`](https://github.com/google/jax/blob/master/docs/jaxpr.rst)? Can't find that test in [`api_test.py`](https://github.com/google/jax/blob/master/tests/api_test.py). Cheers!

**Comment by j-towns** [comment]
Yes, the statement
>(Note: the code examples in this file can be seed also in `jax/tests/api_test::JaxprTest.testExamplesJaxprDoc`.)

is no longer true. @8bitmp3 would you like to submit a pr removing it?
